src/coupling.c

```diff
@@ -320,15 +418,45 @@ int coupling_countOpenings(int j)
 
 //=============================================================================
 
-void coupling_deleteOpening(int j, int idx)
+int coupling_deleteOpening(int j, int idx)
 //
 //  Input:   j = node index
 //           idx = opening index
 //  Output:  none
```

Comment:
Update the comment to fit the new return value.
Thanks for implementing this! That was not trivial.